### ISSUE TITLE
Implement Issues API

### ISSUE THREAD

**Issue by pengwynn** [pengwynn/octonaut#7]
Commands for [Issues API](https://github.com/pengwynn/octokit/blob/master/lib/octokit/client/issues.rb)

**Comment by adamyonk** [comment]
Trying to think through how this should be structured; I can think of three usages:
- A user's issues
- A repo's issues
- Search for issues

Spitballing:
- `$ octonaut user <name> issues`
- `$ octonaut repo <name> issues`
- `$ octonaut issues <query>`

Thoughts?

**Comment by ghost** [comment]
I like that @adamyonk.

**Comment by ghost** [comment]
You have Issues for users and repos but you can also retrieve Issues for Organizations (or would that be a part of the user bit?).

**Comment by adamyonk** [comment]
That's a good question. It would definitely be worth looking into whether you can get at issues from an organizations perspective. Can you speak to that, @pengwynn?

**Comment by pengwynn** [comment]
Yeah I think we'll end up with all three of those. `octonaut org <name> issues`, too.

**Comment by cored** [comment]
Cool stuff, I hope to finish the migration to Octokit 2.0 this week to start working on this new features.
